narwhals/series.py

```diff
@@ -1216,6 +1216,47 @@ def alias(self, name: str) -> Self:
         """
         return self._from_compliant_series(self._compliant_series.alias(name=name))
 
+    def rename(self, name: str) -> Self:
+        """
+        Rename the Series.
+
+        Alias for `Series.alias()`.
+
+        Arguments:
+            name: The new name.
+
+        Examples:
+            >>> import pandas as pd
+            >>> import polars as pl
+            >>> import narwhals as nw
+            >>> s = [1, 2, 3]
+            >>> s_pd = pd.Series(s, name="foo")
+            >>> s_pl = pl.Series("foo", s)
+
+            We define a library agnostic function:
+
+            >>> @nw.narwhalify
+            ... def func(s):
+            ...     return s.rename("bar")
+
+            We can then pass either pandas or Polars to `func`:
```

Comment:
Hey @raisadz thanks for adjusting!

I have one more tiny request: in #1000 we are trying to add pyarrow to the docstrings. Would you mind adding an example with pyarrow as well? Maybe we don't have an example with series yet, but you can create a pyarrow "series" via `s_pa = pa.chunked_array([s])`.

Then the line here would become:

```suggestion
            We can then pass any supported library such as pandas, Polars, or PyArrow:
```
and finally the rendering below 🙃

Reply:
Thanks for the suggestion! I added an example with PyArrow series to the docstrings